command/CommandController.js

```diff
@@ -347,20 +347,22 @@ async function scoutSummaries(userid, jsonBodyAttr, urlAttr, titleAttr, res) {
           let promiseArray = [];
           let arrJson = jsonBody[jsonBodyAttr];
           Object.keys(arrJson).forEach(key => {
-            summaryOptions.uri = summaryLink + arrJson[key][urlAttr];
-            logger.debug('Summary uri is: ' + summaryOptions.uri);
-            promiseArray.push(
-              rp(summaryOptions)
-                .then(sumResults => {
-                  let sumResultsJson = JSON.parse(sumResults);
-                  sumResultsJson['title'] = arrJson[key][titleAttr];
-                  return sumResultsJson;
-                })
-                .catch(function(err) {
-                  logger.error('Caught an error: ' + err);
-                  return JSON.stringify({});
-                })
-            );
+            if (arrJson[key].is_article == '1') {
```

Comment:
Ok, for all of these, we also need to check word_count attribute... in the case of the blockchain article it's 0.  So, if word_count is 0 then it's probably not able to get the content.

Reply:
Actually, cancel that.  We should be able to rely on just the is_article attribute.